app/assets/styles/bulletin.css

```diff
@@ -194,9 +197,45 @@ hr.image {
 	padding: 1em;
 	align-self: center;
 	justify-self: center; 
+}
+
+/* Including PDF ! */
+a[href$=".pdf"]:after { 
+	content: "Read more..."; 
+	text-decoration: none;
+	font-size: 1.10em;
+	color: hsla(240, 100%, 56%, 1);
+}
 
+.btn-small--yellow {
+	background-color: hsla(60, 79%, 60%, 1);
+	border: 2px solid yellow;
+	padding: 7px;
+	border-radius: 5px;
 
 }
+
+.btn-small--yellow:hover {
+	background-color: hsla(210, 100%, 56%, 1);
+	border: 2px solid hsla(210, 100%, 80%, 1);
+	padding: 7px;
+	border-radius: 5px;
+
+}
+
+a[href$=".pdf"]:hover::after {
+	color: white;
+}
```

Comment:
This code should appear after L208 because it helps comprehension in the file. We group like blocks of code close together.

Reply:
I think it is properly placed because it follows the CSS code for  ".btn-small--yellow:hover { }".  The reason is that the rule "color: white" does not work when placed here. However, you are probably right.

Can you please explain, why I cannot change the font color within  ".btn-small--yellow:hover { }" ?

Reply:
Because `.btn-small--yellow` is styling a class and this is a global styling of all `a` elements. Therefor it should be grouped with the other ruleset that is styling all global elements.

Reply:
I see.  But, why should this code be more properly placed after L208 instead of immediately after L222?